scripts/promptflow-ci/promptflow_ci.py

```diff
@@ -165,7 +165,14 @@ def check_flow_run_status(
     parser.add_argument('--flow_submit_mode', type=str, default="sync")
     parser.add_argument('--run_time', type=str, default="default-mir")
     parser.add_argument('--skipped_flows', type=str,
-                        default="bring_your_own_data_qna,template_chat_flow,template_eval_flow")
+                        default="bring_your_own_data_qna,template_chat_flow,template_eval_flow,playground-ayod-rag,"
+                        "chat-quality-safety-eval,qna-non-rag-metrics-eval,qna-quality-safety-eval,"
+                        "qna-rag-metrics-eval,rai-eval-ui-dag-flow,rai-qna-quality-safety-eval,"
+                        "qna-ada-similarity-eval,qna-relevance-eval,ask-wikipedia,classification-accuracy-evals,"
+                        "qna-f1-score-eval,count-cars,analyze-conversations,rerank-qna,multi-index-rerank-qna,"
+                        "detect-defects,analyze-documents,qna-gpt-similarity-eval,use-functions-with-chat-models,"
+                        "qna-groundedness-eval,bring-your-own-data-chat-qna,qna-coherence-eval,template-standard-flow,"
+                        "qna-with-your-own-data-using-faiss-index,chat-with-wikipedia")
```

Comment:
Please also add web-classification and qna-fluency-eval to the skipped_flows, it seems you missed these two.